### DISCUSSION TITLE
Got error when there is chinese character in sql

### DISCUSSION THREAD

**Discussion by license2kill** [reata/sqllineage#639] [Q&A]
Got error when there is chinese character in sql: 
UnicodeDecodeError: 'gbk' codec can't decode byte 0xad in position 8288: illegal multibyte sequence

How can I solve it.

**Comment by twpan-33** [comment]
http://www.360doc.com/content/12/0121/07/81615618_1067006969.shtml

**Reply by twpan-33** [reply]
如果是pycharm的话，可以试试这个哦，我这边同样的问题，解决了
